eng/Versions.props

```diff
@@ -216,7 +216,7 @@
       necessary, and this property is removed from the file.
     -->
     <PrivateSourceBuiltSDKVersion>7.0.104</PrivateSourceBuiltSDKVersion>
```

Comment:
This should be updated as well to 7.0.105.  It requires the SDK from CI being uploaded.